### PR TITLE
[BugFix]fix broker load ut error

### PR DESCRIPTION
**Pull request by zddr**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/util/MasterDaemon.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/common/FeConstants.java|1|1|100.00%|

**Comment by zddr** [comment]
run starrocks_be_unittest

**Comment by zddr** [comment]
run starrocks_be_unittest

**Review by Astralidea — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/common/FeConstants.java [hunk 1]
```diff
@@ -48,6 +48,9 @@ public class FeConstants {
     // set to true to skip some step when running FE unit test
     public static boolean runningUnitTest = false;
 
+    //some ut need this time very long
+    public static long master_daemon_wait_serving_ready_sleep_mills = 10000;
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/common/FeConstants.java:52]: nit: better to have a space in front of the comment

File: fe/fe-core/src/test/java/com/starrocks/analysis/RestrictOpMaterializedViewTest.java [hunk 2]
```diff
@@ -126,6 +127,7 @@ public void testUpdate() {
 
     // This test is temporarily removed because it is unstable,
     // and it will be added back when the cause of the problem is found and fixed.
```
Comment (Astralidea) [inline fe/fe-core/src/test/java/com/starrocks/analysis/RestrictOpMaterializedViewTest.java:129]: if Test annotation is marked should remove upper comment

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/util/MasterDaemon.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/common/FeConstants.java|1|1|100.00%|

**Review by Astralidea — APPROVED** [review]

**Comment by kangkaisen** [comment]
I don't get this PR logic.

**Comment by zddr** [comment]
> 

If the testbrokerload is not completed within 10s, the sleep in masterdaemon is completed (currently 10s), and many tasks continue to be executed downward, jmockit will report the error "missing 1 invocation"

**Review by imay — APPROVED** [review]

**Comment by kangkaisen** [comment]
> > 
> 
> If the testbrokerload is not completed within 10s, the sleep in masterdaemon is completed (currently 10s), and many tasks continue to be executed downward, jmockit will report the error "missing 1 invocation" @kangkaisen

I see. But I don't think this is the best way to fix this issue.

**Comment by kangkaisen** [comment]
> > > 
> > 
> > 
> > If the testbrokerload is not completed within 10s, the sleep in masterdaemon is completed (currently 10s), and many tasks continue to be executed downward, jmockit will report the error "missing 1 invocation" @kangkaisen
> 
> I see. But I don't think this is the best way to fix this issue.  @zddr

**Comment by zddr** [comment]
> > > > 
> > > 
> > > 
> > > If the testbrokerload is not completed within 10s, the sleep in masterdaemon is completed (currently 10s), and many tasks continue to be executed downward, jmockit will report the error "missing 1 invocation" @kangkaisen
> > 
> > 
> > I see. But I don't think this is the best way to fix this issue.  @zddr

I know,but I do not find the better way

**Comment by kangkaisen** [comment]
@zddr I fix this issue in https://github.com/StarRocks/starrocks/pull/8758, you could review it.

**Comment by zddr** [comment]
> @zddr I fix this issue in #8758, you could review it.

3q

**Pull request closed without merging**